<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welScr</class>
 <widget class="QMainWindow" name="welScr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>welScr</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/png/image/welScr.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="startBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>450</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/ico/icon/gameStart.ico);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="referBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>560</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/ico/icon/referBtn.ico);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="quitBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>670</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/ico/icon/playQuitBtn.ico);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>1050</x>
         <y>30</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>华康少女文字W5(P)</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/image/txt.png);</string>
       </property>
       <property name="text">
        <string>登陆</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
